.env.test

```diff
@@ -0,0 +1,8 @@
+HANDLE_TARGET_HOSTNAME=dl.tufts.edu
+HANDLE_EMAIL='"brian.goodmon@tufts.edu"'
+HANDLE_PREFIX=1234.DEV
+HANDLE_ADMIN_PREFIX='"1234.TEST/ADMIN"'
+HANDLE_INDEX=301
+HANDLE_PKEY_FILE=na
```

Comment:
If we're not passing a private_key, we're obviously not registering anything, so why don't the defaults from the sample file here work - i.e. just put in the prod-like hostname in the sample.

Reply:
I'm not sure I follow this, what is the proposal?